### PR TITLE
Create PlatformComponentNonRedHat.json

### PR DESCRIPTION
**Pull request by jaybeeunix**
ServiceLog to warn CUs off of accessing platform components.

### CODE REVIEW

File: osd/PlatformComponentNonRedHat.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Platform component accessed by non-Red Hat person",
```
Comment (aweiteka) [inline osd/PlatformComponentNonRedHat.json:5]: slight better `"Platform component accessed by non-Red Hat user"`

Comment (jaybeeunix) [reply osd/PlatformComponentNonRedHat.json]: changed to "user"

**Review by aweiteka — APPROVED** [review]

**Comment by aweiteka** [comment]
lgtm

**Pull request merged**